src/Statement.py

```diff
@@ -92,6 +93,23 @@ def get_currency(self):
             return self._statement[self._config.get_booking_currency()]
         else:
             return "EUR"
+    
+    @staticmethod
+    def _parse_value(value):
```

Comment:
doc string missing :)

Reply:
Thanks, fixed :)

Reply:
Nice. Thank you. Almost perfect :). The value param description is missing

Reply:
True! Too bad that black does not check for this. I also found nothing in the documentation about adding this check :(